sync/awsdatabricks.py

```diff
@@ -1155,8 +365,8 @@ def write_file(body: bytes):
             else:
                 # Otherwise, update old references and include any new instances in the list
                 newly_added_instance_ids = new_instance_ids.difference(old_instance_ids)
-                updated_instance_ids = newly_added_instance_ids.intersection(old_instance_ids)
-                removed_instance_ids = old_instance_ids.difference(updated_instance_ids)
+                updated_instance_ids = new_instance_ids.intersection(old_instance_ids)
+                removed_instance_ids = old_instance_ids.difference(new_instance_ids)
```

Comment:
Looks like there was a bug here

Reply:
Oops - yep, this changes looks correct to me.